pyqb/__init__.py

```diff
@@ -116,7 +116,7 @@ def __make_req(self, url=None, headers=None, request=None):
         return res
 
     def doquery(self, query=None, qid=None, qname=None, database=None,
-                fields=None, fmt=False, rids=False):
+                fields=None, sort_fields=None, fmt=False, options=False, rids=False):
```

Comment:
@avinashjoshi Can you change this so you don't push rids to the end of the parameters? Otherwise it's changing the contract for people who may not be calling the function with named parameters.

Reply:
Makes sense.